src/zzz_od/application/world_patrol/operation/world_patrol_run_route.py

```
@@ -117,112 +139,192 @@ def run_op(self) -> OperationRoundResult:
-                    # 步长限制调整
-                    sensitivity_change = theoretical_sensitivity - self.sensitivity
-                    clipped_change = max(-0.02, min(sensitivity_change, 0.02))
-                    self.sensitivity += clipped_change
-                    
-                    # 可选：打印调试信息
-                    # log.debug(f"校准: 理论灵敏度={theoretical_sensitivity:.4f}, 新灵敏度={self.sensitivity:.4f}")
-            
-            # 2. 计算并执行本次指令
-            calibrated_angle_diff = angle_diff * self.sensitivity
+        if current_angle is None:
+            # 重置自适应状态，避免使用过时数据
```

Ruff (0.14.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)